rpm/README.md

```diff
@@ -0,0 +1,43 @@
+# System Admin Toolkit Offline Documentation
```

Comment:
@Benjamin-Rohlfing-HPE - thought you might want to review this file, it's documentation specific to the offline format of the docs. Any feedback you have would be welcome.

Reply:
@ekamin-hpe thanks for the ping. Taking a look.